src/test/java/SeleniumTestngTest.java

```diff
@@ -1,24 +1,111 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-// This test class inherits BasicSetupTest class, where the browser is initialized
-// browser variable is available here as it's inherited, so you'll have it available at any place
+import java.util.List;
+
 public class SeleniumTestngTest extends BasicSetupTest {
 
+    private String deleteButton = "button.added-manually";
+
     @Test
     public void abTestingPageHasSpecificTextTest() throws InterruptedException {
-        browser.get("https://the-internet.herokuapp.com/");
-        // Write your code here (just an example provided)
-        WebElement abTestingTaskLink = browser.findElement(By.linkText("A/B Testing"));
-        abTestingTaskLink.click();
+        browser.get("https://the-internet.herokuapp.com/abtest");
+        WebElement abTestingTaskLink = browser.findElement(By.cssSelector("#content > div > h3"));
+        String expectedText = "A/B Test";
+        Assert.assertTrue(abTestingTaskLink.getText().contains(expectedText),
+                "Expected text is not present in the element");
+    }
+
+    @Test
+    public void addRemoveElementsPageVerifyDeleteButton() throws InterruptedException {
+        browser.get("https://the-internet.herokuapp.com/add_remove_elements/");
```

Comment:
Probably would make sense to extract the common URL part into a variable/constant so it can be changed in a single place if needed

Reply:
It makes sense)Thx for your review)